src/applications/pact-act/containers/questions/camp-lejeune/Lejeune-2-4.jsx

```diff
@@ -41,8 +41,8 @@ const Lejeune24 = ({
 
   const locationList = (
     <ul>
-      <li>{RESPONSES.LEJEUNE_MARINE}</li>
-      <li>{RESPONSES.NEW_RIVER_MARINE}</li>
+      <li>Marine Corps Base Camp Lejeune</li>
```

Comment:
I removed the constants for these since we don't usually use constants for `locationList` and this was an oversight.